### PR TITLE
Minor tweaks to README.md

### PR DESCRIPTION
**Pull request by MaVdbussche**
This PR proposes the following changes :

### Reduced reliance on HTML in the README.md file.
Where possible, HTML code blocks have been replaced by their markdown equivalent. I haven't found a solution for the horizontal centering of content though, so ideas there are welcome.

### Very minor formatting tweaks
Some lists had incorrect numbering

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,13 +130,13 @@ docker-compose --env-file ./.env -f docker/docker-compose.build.yml up -d
 Open http://localhost:3333 in your browser and accomplish these steps:
 
 1. Create a new user via _Get Started_ (this first user will get the role `ADMIN`)
-1. Go to the _Market Data_ tab in the _Admin Control Panel_ and click _Gather All Data_ to fetch historical data
-1. Click _Sign out_ and check out the _Live Demo_
+2. Go to the _Market Data_ tab in the _Admin Control Panel_ and click _Gather All Data_ to fetch historical data
```
Comment (dtslvr) [inline README.md:133]: No, it's not incorrect, they get rendered correctly. It's actually a trick that you don't have to modify every item if you change something in the list 🥲

**Review by dtslvr — CHANGES_REQUESTED** [review]
I don't really see a benefit, but it still looks the same 😎 

Can you please revert the numbered list?

File: README.md (hunk 1 above)
Comment (MaVdbussche) [reply README.md:133]: Well I learned something ! :D

**Comment by MaVdbussche** [comment]
Not sure why the checks failed. In particular, not sure about how to get `yarn format:check` to give me a more verbose output to see what exactly I should fix

**Comment by dtslvr** [comment]
> Not sure why the checks failed. In particular, not sure about how to get `yarn format:check` to give me a more verbose output to see what exactly I should fix

I think there are just some invisible control characters. You can locally run `yarn format` to fix them automatically. Then commit the change.

File: README.md (hunk 1 above)
Comment (dtslvr) [reply README.md]: > Well I learned something ! :D

Great! In my opinion, that's the point of open source: to learn new stuff 🤓

**Review by dtslvr — APPROVED** [review]

**Pull request merged**